app/assets/stylesheets/search.scss

```diff
@@ -139,10 +139,8 @@
   height: 180px;
   margin-bottom: 20px;
   object-fit: cover;
-  width: 70%;
-  border-radius: 8px;
-  aspect-ratio: 1/1;
-  object-fit: cover;
+  aspect-ratio: 1 / 1;
+  border-radius: 100%;
```

Comment:
double-check how the border-radius: 100% affects images with varying aspect ratios to ensure consistent styling